src/commands/api/api.ts

```diff
@@ -28,12 +28,13 @@ export const apiCommand = async (apiMethodName: string, options: OptionValues, c
   }
 
   if (!apiMethodName) {
-    error(`You must provide an API method. Run "netlify api --list" to see available methods`)
+    return logAndThrowError(`You must provide an API method. Run "netlify api --list" to see available methods`)
```

Comment:
I didn't remember this about `never`, but TS only seemed to carry over the "unreachable" refinement from the function call's `never` when I `return`.